creator/ingest_runs/queries/validation_run.py

```diff
@@ -1,18 +1,21 @@
-from graphene import relay
+from graphene import relay, Field, String
 from graphene_django.filter import DjangoFilterConnectionField
 from graphql import GraphQLError
 from django_filters import FilterSet, OrderingFilter
 
 from creator.ingest_runs.nodes import ValidationRunNode
 from creator.ingest_runs.models import ValidationRun
+from creator.ingest_runs.common.model import State
 
 
 class ValidationRunFilter(FilterSet):
     order_by = OrderingFilter(fields=("created_on",))
 
```

Comment:
We can add another field here to allow us to filter by a desired set of states:
```python
state = django_filters.CharFilter(lookup_expr=['in'])
```

Note that we can also do this by adding the field directly to the `fields` in the `Meta`:
```
fields = {
    "state": ["in"]
}
```
This is easier, but graphene formats the query parameter in a weird manner where it comes out looking like `state_In`.